src/components/DashBoard.js

```diff
@@ -18,25 +22,72 @@ const gridStyleProperties = makeStyles(theme => ({
   },
   dashBoardProperty: {
     position: 'relative'
+  },
+  legendProperty: {
+    position: 'fixed',
+    width: 210,
+    height: 160,
+    bottom: 10,
+    left: 270,
+    color: 'white',
+    opacity: 0.5,
+    backgroundColor: 'black',
+    borderRadius: 0
+  },
+  legendItemProperty: {
+    padding: '35px 0 10px 0px'
+  },
+  iconProperty: {
+    color: 'red'
   }
 }));
 
+function SinglePoint(props) {
+  return (
+    <SvgIcon {...props}>
+      <path d="M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z" />
+    </SvgIcon>
+  );
+}
+
+function MultiplePoints(props) {
+  return (
+    <SvgIcon {...props}>
+      <path d="M9,7V13H13V17H15V7H13V11H11V7H9M12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22A10,10 0 0,1 2,12A10,10 0 0,1 12,2Z" />
```

Comment:
The look of this 4 is bugging the heck out of me. Can you find a SVG of a 4 in Roboto?